css/index.css

```diff
@@ -130,11 +147,35 @@ img {
 .menu {
   margin-bottom: 20px;
   display: inline-block;
-  margin-left: 10px;
+  margin-left: 5px;
+  padding: 0px;
 }
 /* remove point */
 #email,
 #github {
   list-style: none;
   margin-bottom: 10px;
 }
+
+/*Media Queries to make sure the webpage is responsive for all device sizes (mobile, tablet, desktop, etc.) */
+@media (min-width: 768px) {
```

Comment:
Probably want max-wdith here.  I don't see the display change for a narrow screen.

Reply:
As I understand it, the only thing that needs to be changed to fix the code is change from @media (min-width: 768px) to @media (max-width: 768px). If I misunderstood, please correct me.